Gopkg.lock

```diff
@@ -224,7 +224,6 @@
   revision = "0e69bafe1d790b820cef859dd0126893bdbd53ad"
 
 [[projects]]
-  branch = "master"
```

Comment:
Do you know what's going on here?  I don't understand this removal.  Also, can you spit out the vendor bump from the human-generated changes.  For example, see ac11b74 and a14da04 in #682.

Reply:
@wking ran `dep ensure -update sigs.k8s.io/cluster-api-provider-aws` again to split out the commit and now don't see this change anymore. Not sure why the difference.